src/main/java/uk/gov/hmcts/reform/pip/account/management/controllers/MediaLegalApplicationController.java

```diff
@@ -0,0 +1,127 @@
+
+import java.util.List;
+import java.util.UUID;
+
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+import static org.springframework.http.MediaType.MULTIPART_FORM_DATA_VALUE;
+
+@RestController
+@Api(tags = "Account Management - API for managing media & orphaned legal professional applications")
+@RequestMapping("/application")
+public class MediaLegalApplicationController {
+
+    private final MediaLegalApplicationService mediaLegalApplicationService;
+
+    private static final String NOT_AUTHORIZED_MESSAGE = "User has not been authorized";
+
+    @Autowired
+    public MediaLegalApplicationController(MediaLegalApplicationService mediaLegalApplicationService) {
+        this.mediaLegalApplicationService = mediaLegalApplicationService;
+    }
+
+    @IsAdmin
+    @ApiResponses({
+        @ApiResponse(code = 200, message = "List<{MediaAndLegalApplication}>"),
+        @ApiResponse(code = 403, message = NOT_AUTHORIZED_MESSAGE),
+    })
+    @ApiOperation("Get all applications")
+    @GetMapping(produces = APPLICATION_JSON_VALUE)
+    public ResponseEntity<List<MediaAndLegalApplication>> getApplications() {
+        return ResponseEntity.ok(mediaLegalApplicationService.getApplications());
+    }
+
+    @IsAdmin
+    @ApiResponses({
+        @ApiResponse(code = 200, message = "List<{MediaAndLegalApplication}>"),
+        @ApiResponse(code = 403, message = NOT_AUTHORIZED_MESSAGE),
+    })
+    @ApiOperation("Get all application by the status")
+    @GetMapping(value = "/{status}", produces = APPLICATION_JSON_VALUE)
+    public ResponseEntity<List<MediaAndLegalApplication>> getApplicationsByStatus(
+        @PathVariable MediaLegalApplicationStatus status) {
+        return ResponseEntity.ok(mediaLegalApplicationService.getApplicationsByStatus(status));
+    }
+
+    @IsAdmin
+    @ApiResponses({
+        @ApiResponse(code = 200, message = "{MediaAndLegalApplication}"),
+        @ApiResponse(code = 403, message = NOT_AUTHORIZED_MESSAGE),
+    })
+    @GetMapping(value = "/application/{id}", produces = APPLICATION_JSON_VALUE)
+    public ResponseEntity<MediaAndLegalApplication> getApplicationById(@PathVariable UUID id) {
+        return ResponseEntity.ok(mediaLegalApplicationService.getApplicationById(id));
+    }
+
+    @IsAdmin
+    @ApiResponses({
+        @ApiResponse(code = 200, message = "File returned"),
+        @ApiResponse(code = 403, message = NOT_AUTHORIZED_MESSAGE),
+    })
+    @GetMapping(value = "/image/{id}", produces = MediaType.APPLICATION_OCTET_STREAM_VALUE)
+    public ResponseEntity<Resource> getImageById(@PathVariable String id) {
+        Resource file = mediaLegalApplicationService.getImageById(id);
+
+        return ResponseEntity.ok().body(file);
+    }
+
+    @ApiResponses({
+        @ApiResponse(code = 200, message = "{MediaAndLegalApplication}"),
+    })
+    @ApiOperation("Create a new application")
+    @PostMapping(produces = APPLICATION_JSON_VALUE, consumes = MULTIPART_FORM_DATA_VALUE)
```

Comment:
not sure if we'd want this one locked down too or we could be spammed with bots requesting accounts to the service 😂  @ChrisS1512 do you think this should be locked down also? then only the FE can access this endpoint

Reply:
Just spoke to Chris, I am going to add isAdmin class wide so then the FE can only access that endpoint, originally I thought isAdmin meant the actual user must be admin.